### ISSUE TITLE
Change restart:always to use restart:unless-stopped instead

### ISSUE THREAD

**Issue by mhodge13** [magento/magento2devbox-web#93]
Recently I had an issue with the Magento docker instances not working for some reason.  I then proceeded to restart a new Magento docker instance and it seems I was having issues with that version as well.  I realized that after I ran a "docker ps" that I had multiple (about 20) containers running even after I ran a "docker-compose stop".  The only way that I was able to make the containers stop running, even after restarting docker, was to run a "docker-compose kill" command on the containers.  I should be able to stop and start containers when needed without having to kill the containers every time.  After I reviewed the docker-compose.yml file I noticed that you have the restart set to "always" which will cause the docker containers to "always" restart regardless of restarting docker and/or restarting the computer.  This may not be ideal for someone that is not that familiar with docker and it is not clearly defined why it is setup this way.

My suggestion would be to change the "restart" policy from the current value of "always" to use "unless-stopped".  This allows the containers to be stopped without killing the containers every time.

Example:
`restart:always`
Change to:
`restart:unless-stopped`

OS: Mac
Docker Community Version: `Version 17.03.1-ce-rc1-mac3 (15924)
Channel: edge
1795b358d5`

**Comment by Jrb1x** [comment]
Agreed. For everyone that's already made their containers, you can do this: `docker update --restart=unless-stopped container-name`
